horace_core/sqw/@sqw/compact.m

```diff
@@ -0,0 +1,25 @@
+function wout = compact(win)
+% Squeezes the data range in an sqw or dnd object to eliminate empty bins
```

Comment:
do you mean empty bins at the extrema of the ranges only? does not sound as if internal empty bins get eliminated.

the declaration agrees with this